### ISSUE TITLE
ParseContent() incorrectly advances reader

### ISSUE THREAD

**Issue by GoogleCodeExporter** [barsu8184/google-gdata#42]
```
What steps will reproduce the problem?
1. Parse multiple entries with content elements
2. if (MoveToStartElement(reader) == true) will advance the reader to the 
next entry when it just needs to read the data in the current element

What is the expected output? What do you see instead?
When parsing a feed with 26 entries, only 13 are found and the content of 
the 13 is the entry XML for the other 13.

What version of the product are you using? On what operating system?
1.0.9.3
Windows 2003 SP1

Please provide any additional information below.

```

Original issue reported on code.google.com by `mbecker...@gmail.com` on 19 Jan 2007 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
Can you provide more detail? Like is this text/xml/xhtml content? The best 
would be
to attach a TXT file with the feed content you are trying to parse. 
```

Original comment by `fman...@gmail.com` on 20 Jan 2007 at 7:18
- Changed state: **Accepted**
- Added labels: **Client-CSharp**

**Comment by GoogleCodeExporter** [comment]
```
Attached file
```

Original comment by `mbecker...@gmail.com` on 22 Jan 2007 at 1:35

Attachments:
- [googlenewsfeed.atom](https://storage.googleapis.com/google-code-attachments/google-gdata/issue-42/comment-2/googlenewsfeed.atom)

**Comment by GoogleCodeExporter** [comment]
```
Fixed with version 1.0.9.4 - available via subversion now. Please report back 
if the
issue is not resolved. 
```

Original comment by `fman...@gmail.com` on 25 Jan 2007 at 4:43
- Changed state: **Fixed**
